src/utils.js

```diff
@@ -62,7 +62,10 @@ export function getDefaultRegistry() {
 }
 
 export function getWidget(schema, widget, registeredWidgets = {}) {
-  const { type } = schema;
+  let { type } = schema;
+  if (!type && schema.enum) {
+    type = "string";
+  }
```

Comment:
I'd love to see this duplicated logic extracted to some `getSchemaType` helper function, so we could reuse it in SchemaFiled.js.

Reply:
+1 on this one, can you do that @penx so that we can merge this ?

Reply:
done